docs/book/src/topics/managedcluster.md

```diff
@@ -58,7 +58,7 @@ Execute clusterctl to template the resources, then apply to a management cluster
 
 ```bash
 clusterctl init --infrastructure azure
-clusterctl generate cluster ${CLUSTER_NAME} --kubernetes-version ${KUBERNETES_VERSION} --flavor aks > cluster.yaml
+clusterctl generate cluster ${CLUSTER_NAME} --kubernetes-version ${KUBERNETES_VERSION} --worker-machine-count=${WORKER_MACHINE_COUNT} --flavor aks > cluster.yaml
```

Comment:
while you're here, it looks like the `KUBERNETES_VERSION` in the example is outdated and no longer supported

Reply:
I found a whole bunch of errors with this example and fixed them here: https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2776

FWIW I didn't observe that including `--worker-machine-count=${WORKER_MACHINE_COUNT}` was necessary, clusterctl picks up that env var automatically and does the right thing in the aks flavor template.

Reply:
I mean also `--kubernetes-version` is not necessary, because it is read from the env:

https://github.com/kubernetes-sigs/cluster-api/blob/bda7e5cca7b57767437f3627e8cf2d3d389e173f/cmd/clusterctl/cmd/util.go#L89-L107
I would say even we make both explicit or both implicit in the example